volumes/partner/responses.yml

```
@@ -85,3 +85,21 @@ tablet:
       status_code: 200
       headers: []
       content: "hello world"
+
+    android:
+      status_code: 200
+      headers: []
+      content:
+        tiles:
+          - id: 12349
+            name: 'Example COM'
+            click_url: 'https://example.com/desktop_android?version=16.0.0&key=22.1&ci=6.2&ctag=1612376952400200000'
```

Please update all URLs for this response to contain the correct `form-factor`:

```suggestion
            click_url: 'https://example.com/tablet_android?version=16.0.0&key=22.1&ci=6.2&ctag=1612376952400200000'
```